benjaminbwright/co-lab
# adds user model

❗ Please **DO NOT** open a Pull Request without creating an Issue first. Failure to do so will result in the rejection of the pull request.

## Purpose

Fixes #13 

Running the seed command now executes succesfully. User Model has been created. 

## Changes

adds models folder, containing an index.js and User.js

path/to/file
- models/index.js
- models/User.js


## Steps to Reproduce or Test

Open the database, collab_db on your local machine, run the seeds file to verify it runs without errors
